### PR TITLE
Enable UFMT on test/test_functionalization.py

### PR DESCRIPTION
**Pull request by huniu20**
Part of  #123062
cc @ezyang

### CODE REVIEW

**Comment by ezyang** [comment]
clean

```
diff --git b/test/test_functionalization.py a/test/test_functionalization.py
index 3c1f3c823f2..e79ab0910ef 100644
--- b/test/test_functionalization.py
+++ a/test/test_functionalization.py
@@ -81,6 +81,7 @@ def _functionalize(
     TEST_WITH_TORCHDYNAMO, "https://github.com/pytorch/pytorch/issues/81457"
 )
 class TestFunctionalization(TestCase):
+
     crossref = False
 
     def get_logs(self, func, *inpts, reapply_views=False, run_reinplace=False):
```

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
needs some more whackin

**Comment by ezyang** [comment]
clean

**Comment by huniu20** [comment]
@ezyang Hello, Yang, thank you for reviewing this PR. I don't know how to fix this failure of CI, I would be very grateful if you could give me some advice.

**Comment by ezyang** [comment]
you need to move the noflake pragmas so that they are on the quoted string line, black moved them to the wrong line

File: test/test_functionalization.py [hunk 1]
```diff
@@ -217,7 +268,8 @@ def forward(self, arg0_1):
     view_copy_11 = torch.ops.aten.view_copy.default(view_copy_8, [16, 64, 128, 128]);  view_copy_8 = None
     detach_copy_1 = torch.ops.aten.detach_copy.default(view_copy_11);  view_copy_11 = None
     return detach_copy_1
-    """)  # noqa: B950
+    """,
```
Comment (statelesshz) [inline test/test_functionalization.py:271]:
```suggestion
    """,  # noqa: B950
```

File: test/test_functionalization.py [hunk 2]
```diff
@@ -217,7 +268,8 @@ def forward(self, arg0_1):
...
+        )  # noqa: B950
```
Comment (statelesshz) [inline test/test_functionalization.py:272]:
```suggestion
        )
```

File: test/test_functionalization.py [hunk 3]
```diff
@@ -455,16 +560,20 @@ def forward(self, arg0_1):
     getitem_5 = _fused_moving_avg_obs_fq_helper_functional[5];  _fused_moving_avg_obs_fq_helper_functional = None
     copy_ = torch.ops.aten.copy_.default(arg0_1, getitem_5);  arg0_1 = getitem_5 = None
     return (getitem, getitem_1)
-    """)  # noqa: B950
+    """,
+        )  # noqa: B950
```
Comment (statelesshz) [inline test/test_functionalization.py:564]: Move the noflake pragma so that it is on the quoted string line

**Review by ji-huazhong — APPROVED** [review]
for reference only :)

**Comment by huniu20** [comment]
@ezyang Hi, sorry about the delay of fix noflake issues and thanks for your review

**Comment by huniu20** [comment]
@ezyang Hello, Yang, I don't know what was wrong with the failure of CI. This PR does not involve those three files:
- test/test_proxy_tensor.py
- test/test_torch.py
- test/test_fx.py

BTW, those three tests all passed locally using `python -m pytest test/test_xxx.py`.

**Comment by huniu20** [comment]
Thanks a lot for your kind review 💯

**Pull request closed without merging**